### ISSUE TITLE
Hide all cards from one board in Upcoming Cards

### ISSUE THREAD

**Issue by JuliaSchuhmacher** [nextcloud/deck#4211]
Hi!

We need the option to hide all cards from one board in the board upcoming cards.

**Example:** Since it is possible to link the apps calendar and deck (great!), we use deck not just to schedule tasks, but to create and organise complex events in the calendar (add attachments, discuss, timeline ...) and work collaboratively on these events and keep them in the calendar even after the event took place.

**Problem:** We cannot hide cards from specific boards in the board upcoming cards. As a consequence, all the cards we created in deck to be displayed in the calendar are listed in upcoming cards and as we are not planning to ever delete them, the list of cards being marked as "past due" will be endless sometime. 

**Solution:** When I click on the three dots next to a board, various options appear (board details, edit board ...). I thought I had found the solution when I read "turn off reminder" (sorry if the wording doesn't exactly fit, I try to guess how this is named in the English deck version, we use German), but I was wrong. However, this could be a solution: Add the option "Don't show in Upcoming Cards".

Thank you so much.
Julia

**Comment by schiessle** [comment]
Let me add: same would be really useful for the widget. I also have a lot of shared boards from colleagues, most of them are just "FYI" and shouldn't show up in the widget or upcoming cards. There I only want to see the tasks from my boards / todo lists.

What might be a good addition is to have a exception for a "Don't show in Upcoming Cards" setting to still show upcoming cards of this boards where I'm explicitly assigned too. Similar in Talk where I can disable general notifications but still get notified if someone explicitly mentions me.

**Comment by stefan-niedermann** [comment]
Duplicate of https://github.com/nextcloud/deck/issues/534 in my opinion.

**Comment by schiessle** [comment]
@stefan-niedermann I don't think it is the same. As described by @JuliaSchuhmacher you might have boards which don't contain immediate actions, they are just for tracking ongoing, "never ending" projects. I don't want them to "spam" my upcoming cards and distract myself from the important tasks.

Same is true if you have a lot of shared boards in our organisation, often they are just "for your information" or to look up stuff occasionally. If they are all added to "upcoming cards" it makes the whole overview useless.
